libcontainer/rootfs_linux.go

```diff
@@ -40,7 +40,8 @@ type mountConfig struct {
 // mountEntry contains mount data specific to a mount point.
 type mountEntry struct {
 	*configs.Mount
-	srcFD string
+	srcFD   string
+	idmapFD int
```

Comment:
I'm somewhat curious why we use a string for `srcFD` and an `int` for `idmapFD`. Wondering if we should unify the approach (probably can be done in a follow-up)

Reply:
The srcFD is a string like "proc/self/fd/<int>" and we need it to be like that today. While the idmapFd is just an int, as we will just use it with moveMount later.

There might be some way to unify that, I'm not sure. Wanna create an issue, maybe? :)

Reply:
> The srcFD is a string like "proc/self/fd/" and we need it to be like that today. While the idmapFd is just an int, as we will just use it with moveMount later.

Yup, noticed that 😄 

>  Wanna create an issue, maybe? :)

Sure! Done here; https://github.com/opencontainers/runc/issues/3935

Reply:
Thanks!